diff -u livecd-tools-013.orig/config/livecd-fedora-8-base-desktop.ks livecd-tools-013/config/livecd-fedora-8-base-desktop.ks
--- livecd-tools-013.orig/config/livecd-fedora-8-base-desktop.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-8-base-desktop.ks	2008-08-03 19:44:23.000000000 +0200
@@ -62,7 +62,6 @@
 
 # make sure debuginfo doesn't end up on the live image
 -*debuginfo
-%end
 
 %post
 # FIXME: it'd be better to get this installed from a package
@@ -163,7 +162,6 @@
 # make sure there aren't core files lying around
 rm -f /core*
 
-%end
 
 
 %post --nochroot
@@ -174,4 +172,3 @@
 if [ "$(uname -i)" = "i386" -o "$(uname -i)" = "x86_64" ]; then
   cp /usr/bin/livecd-iso-to-disk $LIVE_ROOT/LiveOS
 fi
-%end
diff -u livecd-tools-013.orig/config/livecd-fedora-8-desktop.ks livecd-tools-013/config/livecd-fedora-8-desktop.ks
--- livecd-tools-013.orig/config/livecd-fedora-8-desktop.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-8-desktop.ks	2008-08-03 19:44:23.000000000 +0200
@@ -126,7 +126,6 @@
 -nss_db
 -vino
 
-%end
 
 %post
 cat >> /etc/rc.d/init.d/fedora-live << EOF
@@ -142,4 +141,3 @@
 
 EOF
 
-%end
diff -u livecd-tools-013.orig/config/livecd-fedora-8-developer.ks livecd-tools-013/config/livecd-fedora-8-developer.ks
--- livecd-tools-013.orig/config/livecd-fedora-8-developer.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-8-developer.ks	2008-08-03 19:44:23.000000000 +0200
@@ -79,7 +79,6 @@
 #@x-software-development
 # I think this is going to be too big on x86_64
 #*-devel
-%end
 
 %post
 # Enable debuginfo repository (useful for frysk, gdb, etc.)
@@ -112,4 +111,3 @@
 ln -s /usr/share/eclipse-demos-0.0.1 "Eclipse demonstration videos"
 popd
 EOF
-%end
diff -u livecd-tools-013.orig/config/livecd-fedora-8-electronic-lab.ks livecd-tools-013/config/livecd-fedora-8-electronic-lab.ks
--- livecd-tools-013.orig/config/livecd-fedora-8-electronic-lab.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-8-electronic-lab.ks	2008-08-03 19:44:23.000000000 +0200
@@ -111,7 +111,6 @@
 #computing
 octave
 
-%end
 
 %post
 
@@ -233,5 +232,4 @@
 /sbin/chkconfig --del rpcidmapd
 /sbin/chkconfig --del rpcbind
 
-%end
 	
diff -u livecd-tools-013.orig/config/livecd-fedora-8-games.ks livecd-tools-013/config/livecd-fedora-8-games.ks
--- livecd-tools-013.orig/config/livecd-fedora-8-games.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-8-games.ks	2008-08-03 19:44:23.000000000 +0200
@@ -152,4 +152,3 @@
 games-menus
 wget
 
-%end
diff -u livecd-tools-013.orig/config/livecd-fedora-8-kde.ks livecd-tools-013/config/livecd-fedora-8-kde.ks
--- livecd-tools-013.orig/config/livecd-fedora-8-kde.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-8-kde.ks	2008-08-03 19:44:23.000000000 +0200
@@ -57,7 +57,6 @@
 python-imaging
 python-reportlab
 
-%end
 
 %post
 # create /etc/sysconfig/desktop (needed for installation)
@@ -95,4 +94,3 @@
 sed -i 's/NoDisplay=true/NoDisplay=false/' /usr/share/applications/liveinst.desktop
 EOF
 
-%end
diff -u livecd-tools-013.orig/config/livecd-fedora-base-desktop.ks livecd-tools-013/config/livecd-fedora-base-desktop.ks
--- livecd-tools-013.orig/config/livecd-fedora-base-desktop.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-base-desktop.ks	2008-08-03 19:44:23.000000000 +0200
@@ -61,7 +61,6 @@
 
 # make sure debuginfo doesn't end up on the live image
 -*debuginfo
-%end
 
 %post
 # FIXME: it'd be better to get this installed from a package
@@ -166,7 +165,6 @@
 # make sure there aren't core files lying around
 rm -f /core*
 
-%end
 
 
 %post --nochroot
@@ -178,4 +176,3 @@
   if [ ! -d $LIVE_ROOT/LiveOS ]; then mkdir -p $LIVE_ROOT/LiveOS ; fi
   cp /usr/bin/livecd-iso-to-disk $LIVE_ROOT/LiveOS
 fi
-%end
diff -u livecd-tools-013.orig/config/livecd-fedora-desktop.ks livecd-tools-013/config/livecd-fedora-desktop.ks
--- livecd-tools-013.orig/config/livecd-fedora-desktop.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-desktop.ks	2008-08-03 19:44:23.000000000 +0200
@@ -127,7 +127,6 @@
 -nss_db
 -vino
 
-%end
 
 %post
 cat >> /etc/rc.d/init.d/fedora-live << EOF
@@ -143,4 +142,3 @@
 
 EOF
 
-%end
diff -u livecd-tools-013.orig/config/livecd-fedora-developer.ks livecd-tools-013/config/livecd-fedora-developer.ks
--- livecd-tools-013.orig/config/livecd-fedora-developer.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-developer.ks	2008-08-03 19:44:23.000000000 +0200
@@ -80,7 +80,6 @@
 #@x-software-development
 # I think this is going to be too big on x86_64
 #*-devel
-%end
 
 %post
 # Enable debuginfo repository (useful for frysk, gdb, etc.)
@@ -113,4 +112,3 @@
 ln -s /usr/share/eclipse-demos-0.0.1 "Eclipse demonstration videos"
 popd
 EOF
-%end
diff -u livecd-tools-013.orig/config/livecd-fedora-electronic-lab.ks livecd-tools-013/config/livecd-fedora-electronic-lab.ks
--- livecd-tools-013.orig/config/livecd-fedora-electronic-lab.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-electronic-lab.ks	2008-08-03 19:44:23.000000000 +0200
@@ -111,7 +111,6 @@
 #computing
 octave
 
-%end
 
 %post
 
@@ -233,5 +232,4 @@
 /sbin/chkconfig --del rpcidmapd
 /sbin/chkconfig --del rpcbind
 
-%end
 	
diff -u livecd-tools-013.orig/config/livecd-fedora-kde.ks livecd-tools-013/config/livecd-fedora-kde.ks
--- livecd-tools-013.orig/config/livecd-fedora-kde.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-kde.ks	2008-08-03 19:44:23.000000000 +0200
@@ -58,7 +58,6 @@
 # save some space
 -autofs
 
-%end
 
 %post
 
@@ -104,4 +103,3 @@
 # workaround to start nm-applet automatically
 cp /etc/xdg/autostart/nm-applet.desktop /usr/share/autostart/
 
-%end
diff -u livecd-tools-013.orig/config/livecd-fedora-minimal.ks livecd-tools-013/config/livecd-fedora-minimal.ks
--- livecd-tools-013.orig/config/livecd-fedora-minimal.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livecd-fedora-minimal.ks	2008-08-03 19:44:23.000000000 +0200
@@ -19,4 +19,3 @@
 authconfig
 rootfiles
 
-%end
diff -u livecd-tools-013.orig/config/livedvd-fedora-kde.ks livecd-tools-013/config/livedvd-fedora-kde.ks
--- livecd-tools-013.orig/config/livedvd-fedora-kde.ks	2008-01-24 17:48:44.000000000 +0100
+++ livecd-tools-013/config/livedvd-fedora-kde.ks	2008-08-03 19:44:23.000000000 +0200
@@ -94,7 +94,6 @@
 # and some extra packages
 koffice-*
 
-%end 
 
 %post
 
@@ -106,4 +105,3 @@
 #DISPLAYMANAGER="KDE"
 EOF
 
-%end 
