diff -urN livecd-tools-17.7.orig/imgcreate/live.py livecd-tools-17.7/imgcreate/live.py
--- livecd-tools-17.7.orig/imgcreate/live.py	2012-04-08 22:59:14.371110268 +0400
+++ livecd-tools-17.7/imgcreate/live.py	2012-04-08 22:59:26.294075962 +0400
@@ -582,9 +582,17 @@
                                            extra = "",
                                            help = "",
                                            index = index))
-
+            linux.append(self.__get_image_stanza(is_xen, isDracut,
+                                           fslabel = self.fslabel,
+                                           isofstype = "auto",
+                                           liveargs = kern_opts,
+                                           long = "^Install " + long,
+                                           short = "liveinst" + index,
+                                           extra = "liveinst",
+                                           help = "",
+                                           index = index))
             if default:
-                linux[-1] += "  menu default\n"
+                linux[-2] += "  menu default\n"
 
             basic.append(self.__get_image_stanza(is_xen, isDracut,
                                            fslabel = self.fslabel,
@@ -596,6 +604,16 @@
                                            help = "Try this option out if you're having trouble starting.",
                                            index = index))
 
+            basic.append(self.__get_image_stanza(is_xen, isDracut,
+                                           fslabel = self.fslabel,
+                                           isofstype = "auto",
+                                           liveargs = kern_opts,
+                                           long = "Start " + long + " into ^RAM.",
+                                           short = "live_ram" + index,
+                                           extra = "live_ram",
+                                           help = "This option for load live system of "+ long+ " into RAM.",
+                                           index = index))
+
             if checkisomd5:
                 check.append(self.__get_image_stanza(is_xen, isDracut,
                                                fslabel = self.fslabel,
@@ -665,8 +683,11 @@
   menu title Troubleshooting
 """
         # Add basic video and check to submenu
-        for b, c in zip(basic, check):
+        #for b, c in zip(basic, check):
+        for b in basic:
             cfg += b
+
+        for c in check:
             if c:
                 cfg += c
 
