diff --git a/docs/livecd-creator.pod b/docs/livecd-creator.pod
index e80a119..2c78892 100644
--- a/docs/livecd-creator.pod
+++ b/docs/livecd-creator.pod
@@ -43,7 +43,7 @@ Defines the file system label. The default is based on the configuration name.
 =item --compression-type=COMPRESSOR
 
 Specify a compressor recognized by mksquashfs. The default is gzip. lzma
-currently requires a custom kernel to produce a functional image. If gzip
+currently requires a custom kernel to produce a functional image. lzo works with 2.6.36+ kernels, but will generally take up more space than using gzip. If gzip
 is used, the -comp option is not passed to mksquashfs to allow the use of
 older versions of mksquashfs.
 
diff --git a/tools/livecd-creator b/tools/livecd-creator
index d1727c5..d352d74 100755
--- a/tools/livecd-creator
+++ b/tools/livecd-creator
@@ -42,7 +42,7 @@ def parse_options(args):
     imgopt.add_option("-f", "--fslabel", type="string", dest="fs_label",
                       help="File system label (default based on config name)")
     imgopt.add_option("", "--compression-type", type="string", dest="compress_type",
-                      help="Compression type recognized by mksquashfs (default gzip, lzma needs custom kernel)",
+                      help="Compression type recognized by mksquashfs (default gzip, lzma needs custom kernel, lzo needs a 2.6.36+ kernel)",
                       default="gzip")
     parser.add_option_group(imgopt)
 
