diff --git a/imgcreate/live.py b/imgcreate/live.py
index e4e0c2e..9153e10 100644
--- a/imgcreate/live.py
+++ b/imgcreate/live.py
@@ -519,7 +519,7 @@ menu hiddenrow 5
                                            fslabel = self.fslabel,
                                            isofstype = "auto",
                                            liveargs = kern_opts,
-                                           long = long,
+                                           long = long + " (Basic Video)",
                                            short = "linux" + index,
                                            basicvideo = basic,
                                            xdriver = xdriver,
